be/src/storage/lake/lake_persistent_index.cpp

```diff
@@ -406,13 +469,9 @@ Status LakePersistentIndex::load_from_lake_tablet(TabletManager* tablet_mgr, con
     _key_size = PrimaryKeyEncoder::get_encoded_fixed_size(pkey_schema);
 
     const auto& sstables = metadata->sstable_meta().sstables();
-    int64_t max_sstable_version = sstables.empty() ? 0 : sstables.rbegin()->version();
-    if (max_sstable_version > base_version) {
-        return Status::OK();
-    }
-    TRACE_COUNTER_INCREMENT("max_sstable_version", max_sstable_version);
-    TRACE_COUNTER_INCREMENT("new_version", metadata->version());
-
+    // Rebuild persistent index from `rebuild_rss_rowid_point`
+    const uint64_t rebuild_rss_rowid_point = sstables.empty() ? 0 : sstables.rbegin()->max_rss_rowid();
```

Comment:
how about the backwards compatibility if the sstable meta is rebuilt from an earlier pb version which doesn't have the max_rss_rowid()

Reply:
Then it will rebuild from begin.

Reply:
And this feature isn't release yet, so we don't have to worry too much about compatibility.